accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -422,6 +427,7 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
               looking to find and fix errors quickly as they are building a
               component, while the Accessibility Assessment view provides
               explanation and suggested solutions for each issue reported.
+              <strong>Note</strong>On rare occasions the Accessibility checker extension does not show up in the developer tools for some non standard sites. The work around is to use a standard site, launch the checker in the developer tools and then load the non standard site.
```

Comment:
```suggestion
              <strong>Note</strong>: On rare occasions the Accessibility Checker extension does not appear in the developer tools for some non-standard sites. The workaround is to go to a standard site, launch the checker in the developer tools and then load the non standard site in the same browser tab.
```

Reply:
Checker extension not loading is the bug of Devtool, It is not because if a site is "standard" or not.